### ISSUE TITLE
Provide init for Machine ID data directory for specific users

### ISSUE THREAD

**Issue by stevenGravy** [gravitational/teleport#22467]
What would you like Teleport to do?

Configure the Machine ID data dir in addition to output dir with `tbot init`.

What problem does this solve?

When running machine id as a non-root user (`teleport` in our examples) the data dir is not initialized by a Teleport process.  That can lead to permission denied on the initial run and requires administrators to setup the data dir.  Also if a Teleport service is running in parallel its default permissions on the `/var/lib/teleport` directory can block access.

If a workaround exists, please include it.

Manually configure the data dir for a `teleport` user to maintain and set executable for the parent dir.

**Comment by strideynet** [comment]
Duplicate of https://github.com/gravitational/teleport/issues/15788
